ketcher-autotests/tests/File-Management/Open-And-Save-Files/Image-files/image-files.spec.ts

```diff
@@ -1771,4 +1771,350 @@ test.describe('Image files', () => {
     await selectEraseTool(page);
     await takeEditorScreenshot(page);
   });
+
+  test('Verify that added from KET SVG images are displayed on preview and can be saved to SVG files with correct positions and layers', async () => {
+    /**
+     * Test case: https://github.com/epam/Indigo/issues/2161
+     * Description: Added from KET SVG images are displayed on preview and saved to SVG files with correct positions and layers
+     */
+    await openFileAndAddToCanvas('KET/svg-images-black-and-colored.ket', page);
+    await selectTopPanelButton(TopPanelButton.Save, page);
+    await clickOnFileFormatDropdown(page);
+    await page.getByTestId('SVG Document-option').click();
+    await expect(page.getByText('Save', { exact: true })).toBeEnabled();
+    await takeEditorScreenshot(page);
+  });
+
+  test('Verify that added by Tool SVG images are displayed on preview and can be saved to SVG files with correct positions and layers', async () => {
+    /**
+     * Test case: https://github.com/epam/Indigo/issues/2161
+     * Description: Added by Tool SVG images are displayed on preview and saved to SVG files with correct positions and layers
+     */
+    await openImageAndAddToCanvas('Images/image-svg.svg', page);
+    await openImageAndAddToCanvas(
+      'Images/image-svg-colored.svg',
+      page,
+      200,
+      200,
+    );
+    await takeEditorScreenshot(page);
+    await selectTopPanelButton(TopPanelButton.Save, page);
+    await clickOnFileFormatDropdown(page);
+    await page.getByTestId('SVG Document-option').click();
+    await expect(page.getByText('Save', { exact: true })).toBeEnabled();
+    await takeEditorScreenshot(page);
+  });
+
+  test('Verify that added from KET SVG images with elements are displayed on preview and can be saved together to SVG file with the correct positions and layers', async () => {
+    /**
+     * Test case: https://github.com/epam/Indigo/issues/2161
+     * Description: Added from KET SVG images with elements are displayed on preview and saved together to SVG file with the correct positions and layers
+     */
+    await openFileAndAddToCanvas('KET/images-svg-with-elements.ket', page);
+    await selectTopPanelButton(TopPanelButton.Save, page);
+    await clickOnFileFormatDropdown(page);
+    await page.getByTestId('SVG Document-option').click();
```

Comment:
Better to create helper function (or at least - create local one) rather than call locator directly (it is used in many tests)

Reply:
Done. I also believe that the code will look more correct this way.